src/styles/styles.js

```diff
@@ -1405,6 +1405,33 @@ function getZoomSizingStyle(isZoomed) {
     };
 }
 
+/**
+ * Returns auto grow text input style
+ *
+ * @param {Number} width
+ * @return {Object}
+ */
+function getAutoGrowTextInputStyle(width) {
+    return {
+        minWidth: 5,
+        width,
+    };
+}
+
+/**
+ * Positions an element to outside of the window and hides it
+ *
+ * @param {Number} windowWidth
+ * @returns {Object}
+ */
+function getHiddenElementOutsideOfWindow(windowWidth) {
+    return {
+        position: 'absolute',
+        opacity: 0,
+        left: windowWidth,
+    };
+}
```

Comment:
Are there any alternatives to this?

Reply:
I wanted to move this hidden element out of the screen so it doesn't block any visible component. Maybe we can try using z-index and move this component background.

Reply:
I'd be interested in seeing if a negative z-index gives us a slightly simpler solution, but I don't think it should block the PR.

Reply:
there is also `transform: translateX()` as an option

Reply:
I used transform solution to hide the element like this:

```
hiddenElementOutsideOfWindow: {
        position: 'fixed',
        top: 0,
        left: 0,
        opacity: 0,
        transform: 'translateX(-100%)',
    }, 
```